### DISCUSSION TITLE
OpenShot Video Editor

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [mayrinck/FOSSonMicrosoftStore#4] [Requests] [answered]
I know that Kdenlive has already been added, but I would personally like to see [OpenShot Video Editor](https://www.openshot.org/) be added to the store too. By the way, what you're doing is absolutely stellar.

Privacy policy: https://www.openshot.org/privacy/

**Comment by mayrinck** [comment, marked as answer]
OpenShot is really awesome. Will be added as soon as possible! 
Thank you for the suggestion.

**Reply by JuliaAgostino** [reply]
You're welcome!

**Comment by mayrinck** [comment]
Hey there @Joshagos, just trying to push this one up, have you found anything new on this one?

**Reply by JuliaAgostino** [reply]
Specifically what new stuff?

**Reply by mayrinck** [reply]
Any new info about it... But as reported on the main page, this one can't be converted :/

**Reply by JuliaAgostino** [reply]
> Any new info about it... But as reported on the main page, this one can't be converted :/

No, there hasn't been any new info about it lately.
